src/API/Metrics/Counter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Metrics;
+
+use OpenTelemetry\Context\Context;
+
+interface Counter
```

Comment:
I also prefer to not having an Interface suffix. 

The PSR naming convention only applies internally to PSR standards. It's (thankfully!) not a standard per se, just saying.
However I agree on being consistent with using the suffix, WHEN the interface is defining behavior.
We have cases where interfaces are just "holders" for constants (for example sem-conv), and they don't need the suffix, since they are not meant to be implemented.